sagemaker-pipelines/nlp/amazon_comprehend_sagemaker_pipeline/sm_pipeline_with_comprehend.ipynb

```diff
@@ -0,0 +1,961 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        "s3://aws-ml-blog/artifacts/comprehend-custom-classification/comprehend-train.csv",
</pre><p>Now that you have copied the data into the customer's default bucket, can you use that version instead of the version in the AWS bucket? (2 occurrences)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2979/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>s3fs (library that allows pandas to read csv from S3) requires AWS credentials to be in environment variables to be able to read from private S3 buckets. I have moved it before copying so that it is clear that these lines are only for inspecting the dataset.</p>